app/algorithms/types.ts

```diff
@@ -0,0 +1,24 @@
+export type Alternative = {
+  _id: string;
+  description: string;
+  election: string;
+};
+
+export type Vote = {
+  _id: string;
+  priorities: Alternative[];
+  hash: string;
+  weight?: number;
```

Comment:
This is optional now. STV will on its own append a weight to each vote at initial processing. NORMAL elections will not do this.